typescript/__tests__/testProgramaticallyCreateConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { AIConfigRuntime } from "../lib/config";
+import { InferenceSettings } from "../types";
+
+describe("test extract_override_settings", () => {
+  // shared state
+  let aiConfigRuntime: AIConfigRuntime = AIConfigRuntime.create(
+    "Untitled AIConfig",
+    undefined,
+    undefined,
```

Comment:
When adding "undefined" as a value, add /*param*/ as a comment on that line to describe which param is undefined.
`/*param*/ description`

Also can you double check if this is correct? Looks like you are setting schemaVersion to undefined? Just set it to "latest"

Reply:
updated and used `latest` as suggested